habitat-baselines/habitat_baselines/config/habitat_baselines/rl/policy/hl_fixed_social_nav.yaml

```diff
@@ -0,0 +1,6 @@
+name: "HierarchicalPolicy"
```

Comment:
Thanks! The reason is that the existing `hl_fixed.yaml` has an `add_virtual_keys` option that overwrites `goal_to_agent_gps_compass` to zero, which is an issue since this sensor is the input of the social nav policy. I checked with Andrew and it is confirmed that we cannot remove this key by using the hydra config overwrite. Hence, `hl_fixed_social_nav.yaml` is created to prevent this. To reduce confusion and without changing the existing yamls, I rename this to `hl_fixed_no_virtual_key.yaml`.